test/quantization/core/test_quantized_tensor.py

```diff
@@ -1032,8 +1037,7 @@ def test_qtensor_masked_fill(self):
                 [numel], scale=scale, zero_point=zero_point, device=device,
                 dtype=qtype)
             # mask fill the whole tensor, equivalent to calling plain vanilla fill
-            mask = torch.tensor(True)
-            torch.tensor(fill_with)
+            mask = torch.tensor(True, device=device)
```

Comment:
does masked_fill_ also work with python scalar? e.g. q_masked_fill.masked_fill_(True, fill_with)

Reply:
I don't think so, `masked_fill` expects a tensor mask, so it will throw a type error.

Reply:
my bad, i got it mistaken for the fill value